spec/factories/contribution.rb

```diff
@@ -1,5 +1,5 @@
 FactoryGirl.define do
   factory :contribution do
-    name 'My first contribution'
+    name 'Deployment'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.